templates/default/fulldoc/html/js/app.js

```diff
@@ -7,44 +7,46 @@ try { sessionStorage = window.sessionStorage; } catch (e) { }
 function createSourceLinks() {
     $('.method_details_list .source_code').
         before("<span class='showSource'>[<a href='#' class='toggleSource'>View source</a>]</span>");
-    $('.toggleSource').toggle(function() {
```

Comment:
Is there a reason code is being replaced if this is theoretically not a breaking API change?

It would seem more safe to test this without any deltas to app.js to ensure that jquery-migrate is in fact 100% compatible.

Reply:
> Is there a reason code is being replaced if this is theoretically not a breaking API change?

The functionality described in toggle breaks, if it's not replaced by click.
You can view ToC which animates hide by default at https://lsegal-yard-jquery-bump-no-edits.netlify.app/

<details>
<summary>Screen recording</summary>

https://user-images.githubusercontent.com/16024985/131713399-a4c58c17-ffcb-49ab-a5e9-5b2e8067a866.mov

</details>

I verified that replacing toggle with click does not break existing functionality even in jQuery 1.7.1
The doc can be viewed at https://lsegal-yard-replace-toggle-with-click.netlify.app/
  * <details>
    <summary>Table of Contents on landing page</summary>

    https://user-images.githubusercontent.com/16024985/131574652-39808355-b4e5-43ba-918d-37459158e1de.mov

    </details>

  * <details>
    <summary>Inheritance tree on Array page.</summary>

    https://user-images.githubusercontent.com/16024985/131574907-bf3d318e-fa61-4201-95ea-020db86875a3.mov

    </details>
  * <details>
    <summary>Source above footer on Array page.</summary>

    https://user-images.githubusercontent.com/16024985/131574943-d3a06add-7ab9-4c24-be15-091e5b51aa59.mov

    </details>
  * <details>
    <summary>Defines on Module: YARD::Handlers::Ruby page.</summary>

    https://user-images.githubusercontent.com/16024985/131583803-f222164c-f1f5-452e-94b9-00f6114f798d.mov

    </details>

Reply:
> The functionality described in toggle breaks, if it's not replaced by click. 

If behavior breaks with `toggle()` then this is considered a backwards incompatible change and can't be pulled into YARD without a major version bump unfortunately. 

Downstream consumers are relying on customized templates that may contain use of toggle that is expected to continue to work.

If this were to be accepted, I believe you would need to backport `toggle()` and any major API changes from jquery. I thought that's what jquery.migrate does?

Reply:
> I thought that's what jquery.migrate does?

I thought the same, but may be the toggle wrapper is incorrect in jquery-migrate.

I'll post a PR with just commit [`56b30fc` (#1397)](https://github.com/lsegal/yard/pull/1397/commits/56b30fc49cbe3031a62a2e962efc3f263d437387) which replaces toggle with click. That way:
* The functionality introduced by toggle works after switching to click in jQuery 1.7.1
* The downstream users of YARD won't be impacted, as jQuery version is not bumped.
* The users who want to upgrade jQuery can replace `jquery.js` vendored by YARD.

Reply:
> I'll post a PR with just commit [`56b30fc` (#1397)](https://github.com/lsegal/yard/pull/1397/commits/56b30fc49cbe3031a62a2e962efc3f263d437387) which replaces toggle with click.

The PR to replace toggle(function, function) with click is posted at https://github.com/lsegal/yard/pull/1399